iOS Client/PodcastsController/PodcastFilterViewController.swift

```diff
@@ -10,11 +10,21 @@ import UIKit
 
 class PodcastsFilterViewController: UIViewController, UITableViewDataSource, UITableViewDelegate, PodcastFilterDelegate {
     @IBOutlet weak var tableView: UITableView!
-    var streams: [PodcastStream] = []
+    var filterOptions: [PodcastFilterOption] = []
+    var visibleOptions: [PodcastFilterOption] = []
```

Comment:
this is a confusing name. Based on the code, this seems to be `enabledOptions`. Visible options implies these are the options present in the table view

Reply:
yeah I'll change to enabledOptions, better name